src/Stan/Report/Html.hs

```diff
@@ -0,0 +1,68 @@
+{-# OPTIONS_GHC -Wno-missing-signatures #-}
+
+{-# LANGUAGE NoOverloadedStrings #-}
+
+{- |
+Copyright: (c) 2020 Kowainik
+SPDX-License-Identifier: MPL-2.0
+Maintainer: Kowainik <xrom.xkov@gmail.com>
+
+HTML to be generated in the report.
+-}
+
+module Stan.Report.Html
+    ( stanHtml
+    ) where
+
+import Clay (render)
+import Data.Char (toLower)
+import Html (body_, div_A, doctype_, footer_, h1_, h2_, head_, header_, html_, main_, meta_A, p_,
+             pre_, style_, title_, ( # ))
+
+import Stan.Report.Css (stanCss)
+
+import qualified Data.List as List (words)
+import qualified Html.Attribute as A
+
+
+stanHtml an = doctype_ # html_ (stanHead # stanBody)
+  where
+    stanBody = body_ (stanHeader # stanMain an # stanFooter)
+
+stanHeader = header_ (h1_ "Stan Report")
+
+stanMain an = main_
+    ( divIdClass "general-info" ""
```

Comment:
I think, for consistency, this class can be named as others, since there's a logic for handling such words, if I understand the code correctly
```suggestion
    ( divIdClass "General Info" ""
```

Reply:
Here I use the id without the header, so I don't put the header at all :slightly_smiling_face: